cfgov/v1/jinja2/v1/landing-page-2/_template-prototype-text-intro-content.html

```diff
@@ -0,0 +1,7 @@
+<h1 class='m-text-introduction_heading'>Text Introduction</h1>
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/4b79432ee367905b603bc84da58c75519f18404a